msm/skill_entry.py

```diff
@@ -327,33 +343,32 @@ def run_pip(self, constraints=None):
         return True
 
     def install_system_deps(self):
+        success = True
```

Comment:
This is probably nitpicky, but conceptually it feels wrong to define success as True by default. 

Could we shift it to an else branch of the `if packages` conditional below? 

So essentially 
```
if packages:
    # try and install
else:
    # there's nothing to install
    success = True
```

Reply:
Sure that will work just as well.

Reply:
Updated the PR